### ISSUE TITLE
Not able to Run-AlValidation without previous app.

### ISSUE THREAD

**Issue by PP1986** [microsoft/navcontainerhelper#1530]
I am attempting to run ALValidation on a new app, but whatever I try, it always comes back with an error concerning  missing previous app info.

**Scripts used**
```
param (
    [Parameter(Mandatory = $true)][string]$licenseFile = $(Read-Host "Input license file path"),
    #[Parameter(Mandatory = $true)][string]$prevAppFile = $(Read-Host "Input appfile location of previous published app"),
    [Parameter(Mandatory = $true)][string]$AppFile = $(Read-Host "Input appfile location of app"),
    [Parameter(Mandatory = $true)][string]$countries = $(Read-Host "Input countries"),    
    [Parameter(Mandatory = $true)][string]$affixes = $(Read-Host "Input used affixes")
)
$validationResults = Run-AlValidation `
    -licenseFile $licenseFile `
    -previousApps ""`  
    -apps $AppFile `
    -validateCurrent `
    -countries $countries `
    -affixes $affixes `
    -supportedCountries $countries
```
Also tried to omit -previousApps completely.

BcContainerHelper is version 1.0.16
Powershell is running as administrator

First part it seems to be correct:
![image](https://user-images.githubusercontent.com/44785533/101764566-7b42e900-3ae0-11eb-8af0-c4b40bdffb6e.png)

But then it ends with this:
![image](https://user-images.githubusercontent.com/44785533/101764262-27380480-3ae0-11eb-8ba7-c5d04225a73f.png)

**Comment by freddydk** [comment]
New version should become available sometime tonight

**Comment by freddydk** [comment]
You can also download the fixed Run-AlCops.ps1 here: https://raw.githubusercontent.com/microsoft/navcontainerhelper/dev/AppHandling/Run-AlCops.ps1
and overwrite the local copy in:
"C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\1.0.16\AppHandling\Run-AlCops.ps1"

**Comment by freddydk** [comment]
Fix shipped in BcContainerHelper 1.0.17-preview282

**Comment by PP1986** [comment]
Thanks!
